src/Controls/tests/TestCases.HostApp/Issues/Issue21980.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+	[Issue(IssueTracker.Github, "21980", "IndicatorView with DataTemplate (custom image) does not render correctly when ItemsSource change.", PlatformAffected.Android | PlatformAffected.iOS)]
+	public partial class Issue21980 : ContentPage
+	{
+		public Issue21980()
+		{
+			InitializeComponent(); 
+			BindingContext = new Issue21980ViewModel();
+		}
+	}
+
+	public class Issue21980ViewModel : INotifyPropertyChanged
+	{
+		private readonly IReadOnlyList<string> _imageSource1 = new List<string>
+		{
+			"https://xcdn.next.co.uk/Common/Items/Default/Default/ItemImages/AltItemZoom/440022s.jpg",
+			"https://xcdn.next.co.uk/Common/Items/Default/Default/ItemImages/AltItemZoom/440022s2.jpg",
+			"https://xcdn.next.co.uk/Common/Items/Default/Default/ItemImages/AltItemZoom/440022s3.jpg",
+			"https://xcdn.next.co.uk/Common/Items/Default/Default/ItemImages/AltItemZoom/440022s4.jpg",
+			"https://xcdn.next.co.uk/Common/Items/Default/Default/ItemImages/AltItemZoom/440022s5.jpg"
```

Comment:
Could these rather be a collection of images in the app, or maybe the dotnet bot multiple times. The network on CI is never reliable enough for testing. No idea why everything else will work but when you need the network for a test it stops working.

Reply:
I will modify the collection of images in the app, or possibly use the dotnet bot image multiple times.